src/main/java/com/example/b2b_opportunities/Service/PositionService.java

```diff
@@ -273,9 +274,14 @@ private RequiredSkill getRequiredSkill(RequiredSkillsDto requiredSkill, Position
         RequiredSkill requiredSkillResult = new RequiredSkill();
         requiredSkillResult.setPosition(position);
         requiredSkillResult.setSkill(skill);
-        if (requiredSkill.getExperience() != null) {
-            Experience experience = experienceRepository.save(ExperienceMapper.toExperience(requiredSkill.getExperience()));
-            requiredSkillResult.setExperience(experience);
+        if (requiredSkill.getMonths() != null) {
+            ExperienceRequestDto experienceDto = new ExperienceRequestDto();
+            experienceDto.setMonths(requiredSkill.getMonths());
+
+            Experience experience = ExperienceMapper.toExperience(experienceDto);
+            experienceRepository.save(experience);
+
+            requiredSkillResult.setMonths(experience.getMonths());
```

Comment:
We need to remove the Experience completely - the requestdto, responsedto, and the entity itself. We dont want to set 'experience' here. We also dont need the experience repository anymore. We just need to set the months field of requiredSkills. Some of the tests are failing because of this.